types/src/on_chain_config/aptos_features.rs

```diff
@@ -157,6 +170,16 @@ impl Default for Features {
         for feature in FeatureFlag::default_features() {
             features.enable(feature);
         }
+        if let Some(enabled_features) = ENABLED_FEATURES.get() {
+            for feature in enabled_features.iter() {
+                features.enable(*feature);
+            }
+        }
+        if let Some(disabled_features) = DISABLED_FEATURES.get() {
+            for feature in disabled_features.iter() {
+                features.disable(*feature);
+            }
+        }
```

Comment:
Only works when running from an empty db, because these can alter the genesis. However in many cases we run benchmark against an existing db. Do we care to hack for that use case?